Packs/AWS-Route53/Integrations/AWSRoute53/AWSRoute53.py

```diff
@@ -164,41 +111,41 @@ def delete_record(args):
             'Status': record['Status']
         })
 
-        output = json.dumps(response['ChangeInfo'], cls=DatetimeEncoder)
-        raw = json.loads(output)
-        ec = {'AWS.Route53.RecordSetsChange': raw}
-        return create_entry('AWS Route53 record deleted', data, ec)
+        output = json.loads(json.dumps(response['ChangeInfo'], cls=DatetimeEncoder))
+        return create_entry('AWS Route53 record deleted', data, output, 'AWS.Route53.RecordSetsChange')
 
-    except Exception as e:
-        return raise_error(e)
+    except Exception as error:
+        return CommandResults(content_format=EntryFormat.TEXT, entry_type=EntryType.ERROR, readable_output=str(error))
```

Comment:
You don't raise an error?
Let's write it to the log then, with all available (sanitized) details.